generators/client/templates/react/src/main/webapp/app/modules/administration/configuration/configuration.tsx

```diff
@@ -40,77 +49,104 @@ export class ConfigurationPage extends React.Component<IConfigurationPageProps,
 
   filterFn = prefix => prefix.toUpperCase().includes(this.state.filter.toUpperCase());
 
+  sortPrefixFn = (prop1, prop2) => {
+    const { configuration } = this.props;
+    if (configuration && configuration.configProps) {
+      const pref1 = this.props.configuration.configProps[prop1].prefix;
+      const pref2 = this.props.configuration.configProps[prop2].prefix;
+      return this.sortFn(pref1, pref2, this.state.reversePrefix);
+    }
+  }
+
+  sortPropertyFn = (prop1, prop2) => this.sortFn(prop1, prop2, this.state.reverseProperties);
+
+  sortFn = (prop1, prop2, reverse) => {
```

Comment:
we already have lodash so can any of the built in sort functions be used instead of writing our own? https://lodash.com/docs/4.17.4